simulation_parameters/microbe_stage/biomes.json

```diff
@@ -25,12 +25,12 @@
                     "name": "Floating Hazard",
                     "meshes": [
                         {
-                            "scenePath": "res://src/microbe_stage/ToxinCloud.tscn"
```

Comment:
This file looks like all the changes are backwards, this reverts the change to moving to using particles for the toxins.